### PR TITLE
Passed all the screen data to complete registration action

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-4682

#### Changes proposed in this Pull Request:

- Passed complete screen data to complete registration action
- Updated userData type

#### Screenshots / Screen Recording (if applicable)
- 
Screenshot 2023-08-30 at 3 21 30 PM

#### To Test:

- cd login-workflow
- yarn start:example
- Complete the registration, keeping console open

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx [hunk 1]
```diff
@@ -115,10 +115,19 @@ export const RegistrationWorkflow: React.FC<React.PropsWithChildren<Registration
     };
 
     const finishRegistration = (data: IndividualScreenData): Promise<void> => {
-        if (actions && actions().completeRegistration)
-            // TODO: THIS LOOKS BROKEN — ARE WE ONLY PASSING THE DATA FROM THE LAST SCREEN OF THE WORKFLOW???
```
Comment (ektaghag-eaton) [inline login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx:119]: Can you check with @daileytj, why he added this comment?

**Review by ektaghag-eaton — APPROVED** [review]
Working fine with custom screen

Screenshot 2023-08-31 at 6 11 06 PM

File: login-workflow/example/src/actions/RegistrationUIActions.tsx [hunk 2]
```diff
@@ -101,11 +101,9 @@ export const ProjectRegistrationUIActions: () => RegistrationUIActions = () => (
         return isRandomFailure();
     },
 
-    completeRegistration: async (
-        userData: any,
-        validationCode: string | number,
-        validationEmail?: string
-    ): Promise<{ email: string; organizationName: string }> => {
+    completeRegistration: async (userData: object): Promise<{ email: string; organizationName: string }> => {
+        // eslint-disable-next-line no-console
```
Comment (ektaghag-eaton) [inline login-workflow/example/src/actions/RegistrationUIActions.tsx:105]: Before merging please remove this line

File: login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx (hunk 1 above)
Comment (daileytj) [reply login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx:119]: I believe that came from @joebochill  during the docs cleanup story

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good, just follow up on comments

File: login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx (hunk 1 above)
Comment (surajeaton) [reply login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx:119]: This PR is to address that todo comment only.

**Pull request merged**